### PR TITLE
*: fix urls cannot take effect when setting in the config file 

### LINKED ISSUE #6190: Fail to start PD when using config file to set urls
### What did you do?
Start PD by using the config file.

### What did you expect to see?
Successfully start

### What did you see instead?
The URLs are not the given ones.

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Fixes #6190.

### What is changed and how does it work?
This PR fixes the problem accidentally introduced by #6004.

```commit-message
```

### Check List

Tests

- Manual test 

config.toml:
```toml
client-urls = "http://127.0.0.1:22379"
peer-urls = "http://127.0.0.1:22380"
```
```sh
bin/pd-server --config=config.toml
```

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: cmd/pd-server/main.go [hunk 1]
```diff
@@ -122,9 +122,9 @@ func addFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolP("config-check", "", false, "check config file validity and exit")
 	cmd.Flags().StringP("name", "", "", "human-readable name for this pd member")
 	cmd.Flags().StringP("data-dir", "", "", "path to the data directory (default 'default.${name}')")
-	cmd.Flags().StringP("client-urls", "", "http://127.0.0.1:2379", "url for client traffic")
+	cmd.Flags().StringP("client-urls", "", "", "url for client traffic")
```
Comment (binshi-bing) [inline cmd/pd-server/main.go:125]:
```suggestion
	cmd.Flags().StringP("client-urls", "", "", "urls for client traffic")
```

File: cmd/pd-server/main.go [hunk 2]
```diff
@@ -122,9 +122,9 @@ func addFlags(cmd *cobra.Command) {
...
 	cmd.Flags().StringP("advertise-client-urls", "", "", "advertise url for client traffic (default '${client-urls}')")
-	cmd.Flags().StringP("peer-urls", "", "http://127.0.0.1:2380", "url for peer traffic")
+	cmd.Flags().StringP("peer-urls", "", "", "url for peer traffic")
```
Comment (binshi-bing) [inline cmd/pd-server/main.go:127]:
```suggestion
	cmd.Flags().StringP("peer-urls", "", "", "urls for peer traffic")
```

**Review by binshi-bing — APPROVED** [review]
LGTM

**Review by nolouch — COMMENTED** [review]
What if 
```
./pd-server 
```
?

**Comment by rleungx** [comment]
> What if
> 
> ```
> ./pd-server 
> ```
> 
> ?

It will use 127.0.0.1:2379 and 2380 as the default value.

**Pull request merged**